stdlib/runtime/gc.gr

```diff
@@ -45,8 +45,18 @@ primitive unbox: Box<a> -> a = "@unbox"
 
 exception DecRefError
 
-export let decimalCount32 = box((n: WasmI32) => 0n)
-export let utoa32Buffered = box((a: WasmI32, b: WasmI32, c: WasmI32) => void)
+let decimalCount32Dummy = (n: WasmI32) => 0n
+let utoa32BufferedDummy = (a: WasmI32, b: WasmI32, c: WasmI32) => void
+
+// When these boxes are backpatched, the reference count of each function will
+// fall to zero which would cause them to be freed. We're not allowed to free
+// anything in runtime mode, so here we prevent that by setting a higher
+// refcount.
```

Comment:
Can you clarify in this comment that this is the case because of `--compilation-mode=runtime`? It wasn't clear when I initially read it

Reply:
Done!